src/universal/modules/team/components/ShortcutsMenu/ShortcutsMenu.js

```diff
@@ -1,108 +1,131 @@
-import React, { Component, PropTypes } from 'react';
-import look, { StyleSheet } from 'react-look';
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
 import FontAwesome from 'react-fontawesome';
-import theme from 'universal/styles/theme';
+import t from 'universal/styles/theme';
+import {srOnly} from 'universal/styles/helpers';
 
-const combineStyles = StyleSheet.combineStyles;
-const shortcutsKeystrokeHeight = '1.5rem';
+const cs = StyleSheet.combineStyles;
+const keystrokeHeight = '1.5rem';
 
-let styles = {};
+let s = {};
 
-@look
-// eslint-disable-next-line react/prefer-stateless-function
-export default class ShortcutsMenu extends Component {
-  static propTypes = {
-    shortcutsList: PropTypes.array,
-    onCloseClick: PropTypes.func
-  };
-
-  renderShortcutMenuItem(shortcut, index, array) {
+const ShortcutsMenu = (props) => {
+  const renderMenuItem = (shortcut, index, array) => {
     let keystrokeStyle = null;
 
     if (index === 0) {
-      keystrokeStyle = combineStyles(styles.shortcutsKeystroke, styles.shortcutsKeystrokeIsFirst);
+      keystrokeStyle = cs(s.keystroke, s.keystrokeIsFirst);
     } else if (index === array.length - 1) {
-      keystrokeStyle = combineStyles(styles.shortcutsKeystroke, styles.shortcutsKeystrokeIsLast);
+      keystrokeStyle = cs(s.keystroke, s.keystrokeIsLast);
     } else {
-      keystrokeStyle = styles.shortcutsKeystroke;
+      keystrokeStyle = s.keystroke;
     }
 
     return (
-      <li className={styles.shortcutsItem} key={index}>
+      <li className={s.shortcutsItem} key={index}>
         <span className={keystrokeStyle}>{shortcut.keystroke}</span>
-        <span className={styles.shortcutsDefinition}>{shortcut.definition}</span>
+        <span className={s.definition}>{shortcut.definition}</span>
       </li>
     );
-  }
+  };
 
-  render() {
-    const { shortcutsList, onCloseClick } = this.props;
+  const {shortcutsList, onCloseClick} = props;
 
-    return (
-      <div className={styles.shortcutsMenu}>
-        <div className={styles.shortcutsLabel}>
-          Keyboard Shortcuts
-        </div>
-        <a className={styles.shortcutsClose} href="#" onClick={onCloseClick} title="Close menu">
-          <FontAwesome name="times-circle" />
-          <span className={styles.shortcutsCloseLabel}>Close menu</span>
-        </a>
-        <ul className={styles.shortcutsList}>
-          {
-            shortcutsList.map((shortcut, index, array) =>
-              this.renderShortcutMenuItem(shortcut, index, array)
-            )
-          }
-        </ul>
+  const handleCloseClick = (e) => {
+    e.preventDefault();
+    onCloseClick();
```

Comment:
since `onCloseClick` is a click hander, it should expect an event, so it should be able to call `e.preventDefault()`.
@ackernaut I know you told me before, but can you remind me why it is best to use an anchor tag for things like this instead of something that doesn't require a `preventDefault`?